tests/tools.bats

```diff
@@ -0,0 +1,97 @@
+load 'test/libs/bats-support/load'
+load 'test/libs/bats-assert/load'
+load 'test/libs/helpers/helpers'
+
+# pre-actions before test cases running
+setup () {
+  # create a temporary folder
+  export TEMP_DIR="/tmp/$(date +%s)"
+  mkdir "${TEMP_DIR}"
+  # create AWS creds stub
+  export test_aws_profile_name="bats_profile"
+  Create-AWS-Creds "${TEMP_DIR}" "${test_aws_profile_name}"
+}
+
+# post-actions after tests complete
+teardown () {
+  # clean-up after tests
+  rm -rf ${TEMP_DIR}
+}
+
+@test "Testig AWS docker container: should be successful" {
+  # Create temp Makefie
+  Create-Makefile "${TEMP_DIR}" "@\$(AWS) \$(arg1)"
+  # Run TEST target 
+  run make TEST \
+      arg1="configure list-profiles" \
+      HOME=${TEMP_DIR} \
+      AWS_PROFILE=${test_aws_profile_name} \
+  # compare result with expected values    
+  assert_success
+  assert_line "${test_aws_profile_name}"
+}
+
+@test "Testig TERRAFORM docker container: should be successful" {
+  TERRAFORM_VERSION="0.13.5"
```

Comment:
Can we use here TERRAFORM_VERSION env var from Makefile?

Reply:
yes, we can, but we need to explicitly in test case specify this version, since it can be changed for tests without the need to change in Makefile